README.md

```diff
@@ -151,6 +164,7 @@ which is used for local run/debug, because the integration tests will remove all
         },
         "fields": {
           "reference": "6KF07542JV235932C",
+          "description": "Jersey Herren Blau L SKU-119982930",
```

Comment:
More probable description would be: `Thank you for your order. Order number: 12345`